.github/workflows/release.yml

```diff
@@ -50,7 +60,9 @@ jobs:
         with:
           config: config.yaml
         env:
-          CR_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          CR_KEY: "${{ secrets.GPG_KEY_ID }}"
+          CR_SIGN: true
```

Comment:
The sign option is also in the config 😊

https://github.com/netbox-community/netbox-chart/blob/9f4b1606cd74519e9e627bb7ad9182672557ff91/config.yaml#L5

Reply:
@LeoColomb Hrm, wonder which one wins. I like the idea of anyone just building for themselves not having to be set up for signing, but CI explicitly enables it.

Reply:
@RangerRick Up to you, but that config is only used by chart-releaser in GitHub Actions, which limits quite a lot the "themselves" use case.
This config is also relatively specific for the repository anyway, so in a way enforcing signing here makes sure non-official builds with cr fail.
https://github.com/netbox-community/netbox-chart/blob/9f4b1606cd74519e9e627bb7ad9182672557ff91/config.yaml#L1-L2

Reply:
Good point, so instead of setting the env var, I should just change the chart to say `true` there.